src/site/stages/build/process-cms-exports/schemas/output/node-person_profile.js

```diff
@@ -6,7 +6,17 @@ module.exports = {
     entityBundle: { enum: ['person_profile'] },
     entityUrl: { $ref: 'EntityUrl' },
     title: { type: 'string' },
-    fieldBody: { type: ['string', 'null'] },
+    fieldBody: {
+      type: ['object', 'null'],
+      properties: {
+        entity: {
```

Comment:
It doesn't seem like the `processed` field is inside of the `entity` object in your [changes](https://github.com/department-of-veterans-affairs/vets-website/blob/5fd597aae7e7bc019e65d53cd98ad17cb00d4188/src/site/stages/build/process-cms-exports/transformers/node-person_profile.js#L19). I think we are good to remove `entity` and have `fieldBody` only checking the `processed` property.

Reply:
This is confusing. I'm running the following command to validate:

`node script/cms/test-bundle-transformers.js --bundle person_profile`

The validation check passes either way:

```js
    fieldBody: {
      type: ['object', 'null'],
      properties: {
        properties: {
          processed: { type: 'string' },
        },
      },
    },
```

or

```js
    fieldBody: {
      type: ['object', 'null'],
      properties: {
        entity: {
          type: 'object',
          properties: {
            processed: { type: 'string' },
          },
        },
      },
    },
```

Only the former is correct, so why doesn't the latter fail validation?

Reply:
That's strange, the latter should be failing validation 🤔. Maybe it's because `entity` isn't being required.